sound/soc/sof/Kconfig

```diff
@@ -258,6 +258,7 @@ config SND_SOC_SOF_IPC3
 
 config SND_SOC_SOF_INTEL_IPC4
 	bool
+	select SND_INTEL_NHLT
```

Comment:
I don't think we should do this @ranj063, because INTEL_NHLT is also dependent on ACPI.
I have an alternate proposal in https://github.com/thesofproject/linux/pull/3581 where a fallback is used to let the compilation proceed.

Reply:
ack @plbossart